src/DataExplorer/DataExplorerVisualizations.jsx

```diff
@@ -53,6 +60,45 @@ class DataExplorerVisualizations extends React.Component {
     );
   }
 
+  exportToWorkspaceCallback = () => {
+    this.setState({toasterOpen: true});
```

Comment:
just for style can you add spaces between the curly braces and text?
`this.setState({ toasterOpen: true });`

Reply:
these can also be combined into one line
`this.setState({ toasterOpen: true, exportedToWorkspace: true });`